### ISSUE TITLE
Decide what to do about Adobe typeface 'hypatia sans semi-bold' used for OpenAustralia strapline

### ISSUE THREAD

**Issue by JoannaHill** [openaustralia/planningalerts#1656]
https://fonts.adobe.com/fonts/hypatia-sans

**Comment by JoannaHill** [comment]
Kat, Matthew and Jo had a good chat to discuss this issue. Jo shared her explorations of open source fonts that could work as a replacement to Hypatia. We landed on Jost. Following which we discussed how lovely Hypatia actually is and did a quick explore of the cost to purchase and use it. We found it is reasonably priced and the license would more than cover our needs. 

Here is Jo's visual exploration.

![Image](https://user-images.githubusercontent.com/17328152/229951763-38e98523-49dd-4426-a0b3-7134693ef683.png)

Here is the font license we think would be suitable. 
https://www.fontspring.com/lic/adobe/desktop
